gulpfile.js

```diff
@@ -355,5 +355,5 @@ gulp.task('prod', function(callback) {
 
 // Runserver for development
 gulp.task('dev:runserver', function(callback) {
-    runSequence('dev', 'lint', 'connect', 'watch', callback);
+    runSequence('dev', 'connect', 'watch', callback);
```

Comment:
@spyshiv I have removed this because I have added it as a dependency of `connect` and if I will also add it here then it will run twice which isn't needed. What do you say?